stores/maker/actions.js

```diff
@@ -0,0 +1,76 @@
+export const SET_LOADING_DONE = '[Maker Page] SET_LOADING_DONE'
+export const MAKER_AUTH_ERROR = '[Maker Page] MAKER_AUTH_ERROR'
+
+export const SET_PAYMENT_DETAILS_COMPLETE = '[Maker Page] SET_PAYMENT_DETAILS_COMPLETE'
+export const SAVE_PAYMENT_DETAILS = '[Maker Page] SAVE_PAYMENT_DETAILS'
+export const SAVE_PAYMENT_DETAILS_DONE = '[Maker Page] SAVE_PAYMENT_DETAILS_DONE'
+export const SAVE_PAYMENT_DETAILS_FAIL = '[Maker Page] SAVE_PAYMENT_DETAILS_FAIL'
+
+export const SET_KYC_COMPLETE = '[Maker Page] SET_KYC_COMPLETE'
+export const SET_LEGAL_COMPLETE = '[Maker Page] SET_LEGAL_COMPLETE'
+
+export const FETCH_APPS = '[Maker Page] FETCH_APPS'
+export const FETCH_APPS_DONE = '[Maker Page] FETCH_APPS_DONE'
+export const FETCH_APPS_FAIL = '[Maker Page] FETCH_APPS_FAIL'
+
+export const SELECT_APP = '[Maker Page] SELECT_APP'
```

Comment:
I've not seen this convention before, in the past typically actions would be of a value such as: `maker-page/SET_LOADING_DONE` -> https://github.com/blockstack/app.co/blob/3cd6b2cf6aaee68905988ef796c322315052ca7b/stores/mining/constants.js

I'm fine with any convention, I just think we should be consistent.

Reply:
Done